File: 프로그래머스/0/181915. 글자 이어 붙여 문자열 만들기/글자 이어 붙여 문자열 만들기.java
/**
* @brief my_string의 index_list의 원소들에 해당하는 인덱스의 글자들을 순서대로 이어 붙인 문자열을 리턴하는 함수
* @param1
* @param2
* @return
*/

class Solution {
    public String solution(String my_string, int[] index_list) {
        String answer = "";
        
        for (int i = 0; i < index_list.length; i++) {
            answer += my_string.charAt(index_list[i]);
        }
        
        return answer;
    }
}

File: 프로그래머스/0/181915. 글자 이어 붙여 문자열 만들기/글자 이어 붙여 문자열 만들기.js
/**
* @brief my_string의 index_list의 원소들에 해당하는 인덱스의 글자들을 순서대로 이어 붙인 문자열을 리턴하는 함수
* @param1 String my_string
* @param2 Integer Array index_list
* @return String answer
*/

function solution(my_string, index_list) {
    var answer = '';
    
    for (i = 0; i < index_list.length; i++) {
        answer += my_string.charAt(index_list[i]);
    }
    return answer;
}